Crane/buildSrc/src/main/java/com/example/crane/buildsrc/Dependencies.kt

```diff
@@ -21,13 +21,14 @@ object Versions {
 }
 
 object Libs {
-    const val androidGradlePlugin = "com.android.tools.build:gradle:4.2.0-alpha08"
+    const val androidGradlePlugin = "com.android.tools.build:gradle:4.2.0-alpha10"
     const val ktLint = "com.pinterest:ktlint:${Versions.ktLint}"
     const val googleMaps = "com.google.android.libraries.maps:maps:3.1.0-beta"
 
     object Accompanist {
-        private const val version = "0.2.0"
-        const val coil = "dev.chrisbanes.accompanist:accompanist-coil:$version"
+        private const val snapshot = "0.2.2.ui-${AndroidX.Compose.snapshot}-SNAPSHOT"
```

Comment:
nit: this is different to how I did it in Owl/Jetsnack i.e. `version = "0.2.2.ui-6824694"`

Don't feel super strongly here but maybe we should align?

Reply:
I'd just hardcode tbh. You'll need to edit it again when we move to alpha03 anyway (since it will be `0.2.2`).

Reply:
Changed! Thanks